<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-E57MT6J" failures="1" tests="8" name="Test: HomePage and Order" time="125.008" errors="0" timestamp="2020-04-03T19:03:12 CEST">
  <testcase classname="tests.HomePageTest" name="test01WomenBtn" time="11.222"/>
  <testcase classname="tests.HomePageTest" name="test02DressBtn" time="8.19"/>
  <testcase classname="tests.HomePageTest" name="test03SameURL" time="18.015"/>
  <testcase classname="tests.Summer_DressesTest" name="test01_clickFirst" time="6.195"/>
  <testcase classname="tests.Summer_DressesTest" name="test02_Order" time="4.305"/>
  <testcase classname="tests.Summer_DressesTest" name="test03_Values" time="9.407"/>
  <testcase classname="tests.Summer_DressesTest" name="test04_registration" time="38.787"/>
  <testcase classname="tests.Summer_DressesTest" name="test05_logInBack" time="8.88">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [2] but found [(empty)]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2] but found [(empty)]
at tests.Summer_DressesTest.test05_logInBack(Summer_DressesTest.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test05_logInBack -->
  <testcase classname="tests.Summer_DressesTest" name="test04_Qty" time="0.0">
    <ignored/>
  </testcase> <!-- test04_Qty -->
  <testcase classname="tests.Summer_DressesTest" name="test06_proceedBtn" time="0.0">
    <ignored/>
  </testcase> <!-- test06_proceedBtn -->
  <testcase classname="tests.Summer_DressesTest" name="test04_register" time="0.0">
    <ignored/>
  </testcase> <!-- test04_register -->
  <testcase classname="tests.Summer_DressesTest" name="test05_Title" time="0.0">
    <ignored/>
  </testcase> <!-- test05_Title -->
</testsuite> <!-- Test: HomePage and Order -->
